<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayList</class>
 <widget class="QWidget" name="PlayList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>361</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Playlist(Drag and drop file to append at the end, double click to play)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:vertical {
    /*border: 1px solid white;*/
    /*background: rgba(255, 255, 255, 80);*/
	background: transparent;
    width: 18px;
    margin: 18px 0 18px 0;
}

QScrollBar::handle:vertical {
    background: rgba(0, 0, 0, 128);
    min-height: 20px;
}

QScrollBar::handle:vertical:hover
{
	background: rgba(0, 255, 0, 128);
}

QScrollBar::add-line:vertical {
    border: 0px solid grey;
    background: rgba(255, 255, 255, 128);
    height: 18px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    border: 0px solid grey;
    background: rgba(255, 255, 255, 128);
    height: 18px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    border: 1px solid grey;
	border-radius: 2px;
    width: 3px;
    height: 3px;
    background: grey;
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(255, 255, 255, 128);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QTableWidget" name="playListTable">
     <property name="styleSheet">
      <string notr="true">* {
	gridline-color: rgba(255, 255, 255, 200);
}
QTableWidget{
	border: none;
	background: rgba(255, 255, 255, 100);
}
QTableWidget::item:selected{
	color: black;
	background: rgba(255, 255, 255, 128);    
}
QHeaderView {
	color: white;
	font: bold 10pt;
	background-color: rgba(255, 255, 255, 80);
	border: 0px solid rgb(144, 144, 144);
	border:0px solid rgb(191,191,191);
	border-left-color: rgba(255, 255, 255, 0);
	border-top-color: rgba(255, 255, 255, 0);
	border-radius:0px;
	min-height:29px;
}

QHeaderView::section {
	color: black;
	background-color: rgba(255, 255, 255, 100);
	border: 3px solid #000000;
	border-radius:0px;
	border-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>235</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>18</number>
     </attribute>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete selected item</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/delete.png</normaloff>:/rc/images/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clear playlist</string>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/clear.png</normaloff>:/rc/images/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Insert file before selected one(in the beginning if none is selected)</string>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/insert.png</normaloff>:/rc/images/insert.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertUrlButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/insert-url.png</normaloff>:/rc/images/insert-url.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Append file at the end</string>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/append.png</normaloff>:/rc/images/append.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addUrlButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Append URL at the end</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/append-url.png</normaloff>:/rc/images/append-url.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setLenFilButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Filter shortest</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/filter.png</normaloff>:/rc/images/filter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid rgb(0, 170, 255);
	border-radius: 2px;
	background: rgba(255, 255, 255, 255);
}

QLineEdit:focus {
	border: 1px solid rgb(0, 170, 255);
	border-radius: 2px;
	background: rgb(230, 239, 255);
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="isCaseSensitive">
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    spacing: 3px;
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
	border: 1px solid rgb(0, 170, 255);
	border-radius: 4px;
}

QCheckBox::indicator:unchecked {
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: rgb(178, 203, 255);
}</string>
       </property>
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/search.png</normaloff>:/rc/images/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchNextButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../hannah.qrc">
         <normaloff>:/rc/images/next.png</normaloff>:/rc/images/next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playListTable</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>insertButton</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources>
  <include location="../hannah.qrc"/>
 </resources>
 <connections/>
</ui>
